docs/reference/pc_reviewing.md

```diff
@@ -0,0 +1,245 @@
+---
+title: "Package Control: Reviewing a package"
+---
+
+<!-- Original: -->
+<!-- https://github.com/wbond/package_control_channel/wiki#reviewing-a-package-addition -->
+
+
+# How we review packages
+
+The main things we guard for:
+
+- Proper use of the formats supported in [`repository.json`][repo].
+- Proper use of dependencies/libraries.
+- Proper use of [messages][msg].
+- Sensible use of shared resources like menus, keybindings.
+- The maintainer being serious about actually maintaining the package.
+
+The following sections contain various procedures
+and things to look for in new or updated packages.
+
+We don't typically test a package.
+As long as the intentions are good,
+the maintainer actively cooperates,
+and the package "plays nice" in the ecosystem,
+whether the package works well is their responsibility.
+
+[msg]: pc_messages.html
+
+
+## 2x2 week timeout
+
+Once we start the review of a package,
+the maintainer is expected to respond in a somewhat timely manner.
+
+After two weeks without a response,
+the maintainer is pinged in the PR
+and the "stale" label is assigned.
+
+After another two weeks without a response,
+close the PR and assign the `timeout` label.
+
+If the author does respond,
+remove the `stale` label
+and start over.
+
+This procedure may not be repeated infinitely!
+We expect authors to be serious about their package
+and that they intend to maintain it.
+
+
+## Squash all the things
+
+When merging a PR use the "squash and merge" option.
+
+
+## A new package
+
+The [submission guide][submit] must be followed.
+Additionally, we check for the following:
+
+[submit]: pc_submitting.html
+
+
+### The PR itself
+
+* Check if the tests are passing.
+  Make the PR author aware of any issues.
+  There may be scenarios
+  where the tests are incomplete,
+  so always review the changes briefly yourself.
+
+* The PR should contain no changes
+  beyond the targeted package(s)
+  (e.g. formatting the entire file).
+
+* Branch-based versioning is deprecated.
+  Tags must be used in all scenarios.
+
+* Entries should only set the properties they need.
+  See the [`repository.json` documentation][repo] for details.
+ 
+* Search for similar packages.
+
+* Confirm that the user who authors the pull request
+  has push access to the repo they want to add.
+  For an organization,
+  check if the user is part of that organization
+  or check if they pushed commits to the repo.  
+  If the user does not have access,
+  verify that the repo's owner has given permission.
+
+
+### The package
+
+* A README is required,
+  otherwise people won't know what the package does,
+  and should be understandable for new users.
+  It should probably also be in English,
+  except where it obviously shouldn't.
```

Comment:
```suggestion
* A README is required
  and should be understandable for new users.
  Otherwise people won't know what the package does.
  It should probably also be in English,
  except where it obviously shouldn't.
```